.github/ISSUE_TEMPLATE/cm---issue-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: CM - Issue template
+about: Bug report
+title: [BUG] Please provide a title 
+labels: 'bug'
+assignees: ''
+
+---
+
+I'm submitting a bug report.
+
+## What is the current behaviour?
+> Explain what the issue is including screenshots and/or links. What did you do to get this issue?
+
+
+## What is the expected behaviour?
+> What is supposed to happen when the bug is fixed?
+
+
+## Do you know what cause the issue? If yes, can you list or briefly explain.
+> Explain the cause of the issue
+
+
+## Solution (or options to fix the bug)
+
+- [ ] Step 1
+- [ ] Step 2
+- [ ] Step 3
+
+[Link to Trello card](link)
```

Comment:
I think we should not require a bug reporter to link bug issue to an existing Trello card.

IMHO:
- A team member should first triage a bug issues raised in this git repo, and confirm that the bug is a valid one.Triager should confirm with dev lead if bug should be prioritised for work. 
- If the bug fix is known and is a quick fix, and submit a PR and follow PR review process.
- If the bug fix is more involved, we can either create a new trello card to prioritise the work to fix it, or link it to an existing trello card.

what do you think @OlaShabalina ?